test-automation-school/src/main/java/test/automation/school/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,60 @@
+package test.automation.school.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Created by Max on 5/17/2017.
+ */
+public class SearchPage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @FindBy(xpath = "//*[@name='btn'] [@value='Искать' or @value='Meklēt']")
```

Comment:
In my opinion, finding elements by text should be considered as a last resort - text values often tend to change and what if there will be multiple language support, but that`s not always the case. There is id for the element - "sbtn" and it doesnt look dynamic one, but i can be wrong. Again, this should be communicated with the Frontend devs.